aries-site/src/pages/learn/the-box-model-part-one.mdx

```diff
@@ -0,0 +1,227 @@
+import { Box, Image } from 'grommet';
+
+In this tutorial you will:
+
+- Leverage various Box properties to build a hero section for a page.
+
+At the end of this tutorial, you will have built this:
+
+<Box width="large">
+  <Image src="/learnImages/box-model-part-one/box-model-final.png" />
+</Box>
+
+## Prerequisites
+
+This tutorial can be completed by following the steps below but introductory knowledge of [React](https://reactjs.org/docs/getting-started.html) will help ensure the details of the syntax and concepts make sense.
+
+## Getting started
+
+Let’s get started. Use this [CodeSandbox template](https://codesandbox.io/s/hpe-design-system-tutorial-starter-be7kse?file=/src/App.js) as a starter.
+
+First, add the Box to your project by importing it from Grommet and add some placeholder text like “Hello world.”
+
+```
+import { Box, Grommet } from 'grommet';
+import { hpe } from 'grommet-theme-hpe';
+...
+return (
+    <Grommet theme={hpe} full="min">
+	    <Box>Hello world</Box>
+    </Grommet>
+)
+```
+
+## Add hero section content
+
+Now, you'll add the content within the Box. Start by importing Button, Heading, Image, and Text.
+
+```
+import { Box, Button, Heading, Image, Text } from 'grommet';
+```
+
+Delete the placeholder “Hello world” and add in the following content as children of the Box.
+
+To learn more about the properties applied to these components, check out the API docs for [Heading](https://v2.grommet.io/heading), [Text](https://v2.grommet.io/text), and [Button](https://v2.grommet.io/button).
+
+```
+...
+<Box>
+  <Heading margin="none">
+    New HPE ProLiant Servers Engineered for your Hybrid World
+  </Heading>
+  <Text size="xlarge">
+    With a cloud operating experience, built-in security, and optimized
+    performance for your workloads to drive your business forward.
+  </Text>
+  <Button label="Learn more" kind="cta-primary" />
+  <Image src="https://www.insight.com/content/insight-web/en_US/shop/partner/hewlett-packard-enterprise/product-portfolio/servers/jcr%3Acontent/top-inner-full-width/column_layout/-column-1/image_0.img.jpg/1566699651505.jpg" />
+</Box>
+...
+```
+
+At this point, your project should look something like this:
+
+<Box width="large">
+  <Image src="/learnImages/box-model-part-one/step-1.png" />
+</Box>
+
+## Use direction to affect content layout
+
+Now, we’d like to adjust the layout direction of the content in the Box. To do so, apply `direction="row"` to the Box. By default, the layout direction of Box is “column”.
+
+```
+...
+<Box direction="row">
+  <Heading margin="none">
+...
+```
+
+Notice how all the content is displayed in a row because each element was a direct child of the Box we added `direction` property to. However, we would like the heading, text, and button to be grouped and displayed in a column. To do so, wrap these elements in a Box of their own.
```

Comment:
```suggestion
Notice how the content is displayed in a row from left to right. That is because each element was a direct child of the Box to which we added the `direction` property. However, we would like the heading, paragraph, and button to be grouped and displayed in a column. To do so, wrap these elements in a Box of their own.
```